public/doc/openapi-3-v1.0.0.json

```diff
@@ -3243,6 +3318,23 @@
           }
         }
       },
+      "ServicePlansCollection": {
+        "type": "object",
+        "properties": {
+          "meta": {
+            "$ref": "#/components/schemas/CollectionMetadata"
+          },
+          "links": {
+            "$ref": "#/components/schemas/CollectionLinks"
+          },
+          "data": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ServicePlan"
```

Comment:
@lindgrenj6 where is this defined, is it just an object or are we going to be explicit about the properties or would be using the json schema for it

Reply:
It's defined here for us: https://github.com/ManageIQ/catalog-api/blob/6f6f8bcf250f7aec244379fdd93386e0a7d5a2d7/public/doc/openapi-3-v1.0.0.json#L3174-L3201

It's just the internal representation of how we're storing it in the DB. The ultimate goal is that we're going to be serving it through `/service_plans/:id/base` that will return the `base` attribute and the same for `modified`, the only time it'll return the entire schema will be when getting both schemas from the portfolio_item.

It's using the same schema for `base` and `modified` as Topolgy uses for the `create_json_schema` part of the service plan since that is what we're importing: https://github.com/lindgrenj6/topological_inventory-api/blob/a0d5b9adca0633764a4a10edb6b4e60cc486b445/public/doc/openapi-3-v1.0.0.json#L7623-L7626